src/servers/Http1Server.cc

```diff
@@ -394,12 +392,14 @@ Http::One::Server::noteTakeServerConnectionControl(ServerConnectionContext serve
 ConnStateData *
 Http::NewServer(const MasterXaction::Pointer &xact)
 {
+    debugs(11, DBG_PROTOCOL, "switch to HTTP on: " << xact->tcpClient);
```

Comment:
Please remove this line. This change is controversial (e.g., I am not sure we are switching to HTTP _here_ and would need to develop a clear notion of protocol scope to be certain) and lies outside the declared PR scope.

Reply:
An HTTP server AsyncJob is being created and assigned ownership of the connection (per your definition of Comm::Connection lifecycle vs ownership discussions). This should not be controversial.